docs/data/routes/docs/nextjs/tracking-and-analytics/configuration/en.md

```diff
@@ -0,0 +1,66 @@
+---
+name: configuration
+routeTemplate: ./data/component-templates/article.yml
+title: Enabling tracking and analytics
+---
+# Walkthrough: Enabling tracking and analytics
+
+This topic will guide you through the steps required to enable full Sitecore tracking and analytics for your Next.js application.
+
+> Note this does not cover client-side tracking via the JSS tracking API, which is possible for *both SSG and SSR*. Please see the [JSS Tracking](/docs/fundamentals/services/tracking) page for details.
```

Comment:
This makes me think "So what is the difference between client-side tracking and this tracking?"

Reply:
Reworked this to hopefully make it more clear.